spec/models/achievement_spec.rb

```diff
@@ -39,6 +39,17 @@
     end
   end
 
+  describe "to_s" do
+    it "is well-formed" do
+      subject.side = 'corp'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.